src/Microsoft.AspNet.Razor.Runtime/TagHelperContent.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Abstract class which is used to buffer content returned by <see cref="TagHelpers.ITagHelper"/>.
+    /// </summary>
+    public abstract class TagHelperContent
```

Comment:
Why not an interface here? The base type doesn't seem to have have any functional code